Core/Object Arts/Dolphin/Base/Message.cls

```diff
@@ -107,6 +107,32 @@ setSelector: aSymbol arguments: argArray
 	selector := aSymbol.
 	args := argArray!
 
+storeOn: aStream
+
+	self argumentCount == 0 ifTrue: [
+		aStream
+			nextPutAll: '(Message selector: ';
```

Comment:
As with the selectors, emit them directly from the object itself so that the reference is visible to tooling, rather than using a string.